themes/theme-ios11/widgets/Html/index.jsx

```diff
@@ -101,9 +101,19 @@ class Html extends Component {
   handleTap = (e) => {
     const aTag = e.target.closest('a');
 
+    const {
+      attributes: {
+        href: { value: href = '' } = {},
```

Comment:
It seems that the precent encoded spaces from the query parameters get lost at some point. Such urls now contain spaces, and can't be handled by the `openPageExtern` command on iOS.